### ISSUE TITLE
New feature, rooms spaces for let the users join them.

### ISSUE THREAD

**Issue by Adnanhmd** [lovelle/jquery-chat#21]
Hi

I have one server and in that server there are multiple projects. Is there is anyway to use the chat application without colliding different users of different projects. And all should be done only in one server start or from one IP:Port.

**Comment by lovelle** [comment]
Sorry but I don't quite understand what you want to do, you can listen the server chat in any port and all clients can connect to that port, so in that scenario you have all in the same host.

**Comment by Adnanhmd** [comment]
Hi

Let me be more clear, I have one server called "abc"
In that server "abc", i have three different projects a1,b1 and c1.
I want to use chat application in all three different projects a1,b1 and c1. But i don't want a1 users to see b1 users and b1 users to see c1 and vice versa. I know it can be done by running the sever.js in three different port. But is there anyway to use that chat application using one port. Since i don't want to run three different node server.js files.

**Comment by lovelle** [comment]
You are totally right, I think the best way to do this is letting the users to join in different room.
This is an interest feature to have, I will let this issue open in order to do this new feature ASAP.

**Comment by AlessandroChecco** [comment]
It seems the chat room has been implemented in the live version...how can I implement it?
